site/src/components/helpers/PageMeta.js

```diff
@@ -39,7 +50,7 @@ function PageMeta(props) {
         {/* META OPEN GRAPH / FACEBOOK */}
         <meta property="og:title" content={title} />
         <meta property="og:type" content="website" />
-        {/*<meta property="og:url" content={url}/> */}
+        <meta property="og:url" content={'https://medialab.sciencespo.fr'}/>
```

Comment:
Bah alors? On aime pas les `"`?

Reply:
https://github.com/medialab/website/pull/378/commits/753782396b7787a1698c43f6c06011e93761930e for your eyes only